### PR TITLE
Preprocess filepath to allow user to use tilde character for the home

### PR DESCRIPTION
**Pull request by kingzbauer**
I've been looking forward to have such a cli app for myself. Great work.
I realised I cannot use the `~/` character for the home directory. Provided minimal change for that. Kindly have a look

### CODE REVIEW

**Comment by manojkarthick** [comment]
Thanks a lot for the PR @kingzbauer ! - we have [go-homedir](https://github.com/mitchellh/go-homedir) as a dependency that has a `homedir.Expand()` method that provides this functionality. Do you think we re-use that method instead?

**Comment by kingzbauer** [comment]
Certainly!

**Comment by kingzbauer** [comment]
Just realised that this is best done when loading the configuration as opposed to doing this at the call site. So I'll update this shortly with that.

**Comment by manojkarthick** [comment]
Thanks, was about to suggest that - you beat me to it!

File: cmd/root.go [hunk 1]
```diff
@@ -122,7 +122,21 @@ func initConfig() {
...
+		if !config.DisableDb {
+			config.DbName, err = homedir.Expand(config.DbName)
+			if err != nil {
+				logger.Fatalf("Unable to process the database file path: %v", err)
+			}
+		}
+		if !config.DisableCSV {
+			config.CsvName, err = homedir.Expand(config.CsvName)
+			if err != nil {
+				logger.Fatalf("Unable to process the database file path: %v", err)
```
Comment (manojkarthick) [inline cmd/root.go]: Maybe update the error message to `Unable to process the CSV file path`?

**Comment by manojkarthick** [comment]
Thank you @kingzbauer ! I will make a new release shortly.

**Comment by manojkarthick** [comment]
Released v0.2.1: https://github.com/manojkarthick/expenses/releases/tag/v0.2.1

**Comment by kingzbauer** [comment]
Awesome. I'll keep testing the app. Thank you!

**Pull request merged**